terraform/modules/baseline/variables.tf

```diff
@@ -1021,3 +1021,9 @@ variable "resource_explorer" {
   type        = bool
   default     = false
 }
+
+variable "cost_usage_report" {
+  description = "Enables AWS Cost Usage Report"
+  type        = map(any)
```

Comment:
Best to spell out the structure of the variable like we do in other variables

Reply:
Done